src/components/TopBar.js

```diff
@@ -226,6 +227,19 @@ export default _.flow(
             ]),
             'Your Workspaces'
           ]),
+          linkToJobManager && h(Clickable, {
+            as: 'a',
+            target: '_blank',
+            style: styles.nav.item,
+            hover: { backgroundColor: colors.darkBlue[1] },
+            href: 'https://jobs.terra.bio/', //unfiltered view of job manager link
+            onClick: () => this.hideNav()
+          }, [
+            div({ style: styles.nav.icon }, [
+              icon('layers', { className: 'is-solid', size: 24 })
+            ]),
+            'Your Jobs'
```

Comment:
Should this have a "external link" icon next to it, like the Free Credits link does in the left sidebar? It is opening a new window, but it isn't external to Terra like the other link is.